pkg/render/node.go

```diff
@@ -992,6 +995,12 @@ func (c *nodeComponent) nodeMetricsService() *v1.Service {
 					TargetPort: intstr.FromInt(int(nodeReporterPort)),
 					Protocol:   v1.ProtocolTCP,
 				},
+				v1.ServicePort{
```

Comment:
Does the comment on this function apply to this port also? I think the answer is yes but just wanted to call that out.

Reply:
Thanks for catching that. Fixed now.